docs/DesignDocs/ExtendedEnclaveInitializationData.md

```diff
@@ -25,8 +24,7 @@ User Experience
 
 The new API is experimental and details are subject to change, but the examples given here give an idea of the added functionality.
 
-We recommend that all parameters specified at enclave signing time should be set to a value that would prevent the enclave from loading, typically 0. In the case of SGX that means `NumStackPages=0`, `NumHeapPages=0`, and `NumTCS=0`.
-This guarantees that enclave images meant to be used with EEID cannot be accidentally initialized with traditional attestation.
+We recommend that all parameters specified at enclave signing time should be set to a value that would prevent the enclave from loading, typically 0. This guarantees that enclave images meant to be used with EEID cannot be accidentally initialized with traditional attestation. In the case of SGX that means `NumStackPages=0`, `NumHeapPages=0`, and `NumTCS=1`. `NumTCS=0` would be a more obvious choice than `NumTCS=1`, and simplify the EEID report verification implementation, but based on the specific existing implementation of TCS page initialization in OE SDK, `NumTCS=1` is chosen for a security consideration. The rational is discussed in later section about EEID reports verification.
```

Comment:
Line is too long, please split.  Also a typo, "rational" should be "rationale".  Also 'EEID report verification" should be "EEID evidence verification".  (Github for some reason is no longer showing be the suggested text option)